aptos-move/framework/aptos-framework/sources/configs/Stake.move

```diff
@@ -292,7 +295,7 @@ module AptosFramework::Stake {
         while (i < len) {
             let d = Vector::borrow_mut(v, i);
             let reward = TestCoin::zero(); // mint some coins based on delegation, timestamp, maybe also total stakes
-            TestCoin::merge(&mut d.coins, reward);
+            TestCoin::merge(&mut d.rewards, reward);
```

Comment:
weird naming.. why is one of them plural..

Reply:
one has multiple rewards!